### ISSUE TITLE
Modify CheckEnv.js to notify dev if Vite keys are missing from .env file

### ISSUE THREAD

**Issue by aqandrew** [hackforla/311-data#1824]
### Overview

We need to modify our `check-env` script to notify the dev if they are using non-vite compatible environment variables so that we can ensure previous devs have migrated to Vite.

### Action Items

- [x] check keys with checkEnv.js and determine if VITE_ prefix exists, add the prefix if it is not present
- [x] notify the dev if the prefix was added

### Resources

Info about relevant files
- in package.json
  -  start command: ` "start": "npm run check-env && npm run dev",`
  -  check-env command: `"node ./utils/checkEnv"`
- checkEnv.js
  - `utils/checkEnv.js`

### Archived Info

<details><summary>Details</summary>
<p>

> 
> ### Overview
> 
> This change is a simple variable renaming, which must be done for everyone running 311-Data after we switch from webpack to Vite. It could be done manually by inserting `VITE_` at the beginning of their .env file. Or we could automate it, for example by modifying [checkEnv.js](https://github.com/hackforla/311-data/blob/d68ff874b7433344d26fe0f92865a9502e700cc8/utils/checkEnv.js).
> 
> Additionally, the repository secret must be updated so that the Mapbox API key is available in GitHub Actions. See https://github.com/hackforla/311-data/issues/1778#issuecomment-2316193782
> 
> ### Action Items
> - [x] determine the best method for renaming this variable for all 311-Data developers (manual or using a script)
> - [ ] ensure that this variable is renamed for everyone by the time #1822 is merged
> - [ ] add/update the correctly-named repository secret by the time #1822 is merged
> 
> ### Resources/Instructions
> None

</p>
</details>

**Comment by traycn** [comment]
Reviewed [PR #1822](https://github.com/hackforla/311-data/pull/1822#pullrequestreview-2304043821) dependency and it's looking great. Appreciate the clean up in the DbProvider.

**Comment by traycn** [comment]
For existing developers, modifying the [checkEnv.js](https://github.com/hackforla/311-data/blob/d68ff874b7433344d26fe0f92865a9502e700cc8/utils/checkEnv.js) file sounds like a great idea. It'd be nice to take into account those previous devs who aren't aware of this change too.

For new developers, we can update the `.example.env` file and pre-pend `VITE_` for the env variables there. So that `cp .example.env .env` reflect these changes. Refering to the command in the ([README.md](https://github.com/hackforla/311-data)) steps.

**Comment by aqandrew** [comment]
Updating `.example.env` is a great idea. I'll update/test checkEnv.js and include that in the PR as well.

**Comment by ryanfchase** [comment]
@traycn @aqandrew I'm going to modify the ticket in a couple of different ways:

1. dev leads are going to take care of this in a separate issue:
- [ ] ensure that this variable is renamed for everyone (they should all modify `.env` file since this is not checked into git)

2. regarding automation of checking `VITE_` prefix, we will make a dev ticket to update checkEnv.js to throw error (e.g. "Vite prefix needed in `.env` file")
- [ ] I'd like to change the title / overview of _this_ ticket to accomplish this change

**Comment by ryanfchase** [comment]
This ticket is now dedicated to modifying the CheckEnv.js file. Changes to .example.env will be done on [this ticket's](https://github.com/hackforla/311-data/issues/1778) [PR](https://github.com/hackforla/311-data/pull/1822)

**Comment by ryanfchase** [comment]
After discussing this ticket, we've decided this work should be done as part of the existing [PR](https://github.com/hackforla/311-data/pull/1822). @aqandrew will get started on this, the work needs to be completed before the PR is merged.

**Comment by ryanfchase** [comment]
Moved to In Review since I've been re-requested on the [PR](https://github.com/hackforla/311-data/pull/1822).
